botocore/response.py

```diff
@@ -78,9 +79,12 @@ def set_socket_timeout(self, timeout):
             # in py2 and py3.  So this code has been pushed to botocore.compat.
```

Comment:
Might be good to do a once over some of the in-code comments now that a lot of py2/py3 stuff has been removed.